tests/model/generated/endpoint/test_monetary_account_joint.py

```diff
@@ -37,10 +37,10 @@ def test_monetary_account_joint_parser(self):
         with open(file_path, self._FILE_MODE_READ) as f:
             json_string = f.read()
 
-        joint_account: MonetaryAccountJoint = MonetaryAccountJoint.from_json(
+        joint_account = MonetaryAccountJoint.from_json(
             json_string
         )
-
+        self.assertIsInstance(joint_account, MonetaryAccountJoint)
```

Comment:
please add a new line before the assertion. It is better for readability to group assertions together.